packages/js/product-editor/package.json

```diff
@@ -51,9 +51,10 @@
 		"@wordpress/data": "wp-6.0",
 		"@wordpress/date": "wp-6.0",
 		"@wordpress/deprecated": "wp-6.0",
-		"@wordpress/editor": "wp-6.0",
 		"@wordpress/edit-post": "wp-6.0",
+		"@wordpress/editor": "wp-6.0",
 		"@wordpress/element": "wp-6.0",
+		"@wordpress/hooks": "wp-6.0",
```

Comment:
I think the `pnpm.lock` should still get updated?
At-least when I ran `pnpm i` it updated my `pnpm-lock` locally whit the `@wordpress/hooks` addition.

Reply:
👍 Updating...